easybuild/easyconfigs/r/RCall/RCall-0.13.17-disable_gpu_test.patch

```diff
@@ -0,0 +1,32 @@
+diff -ruN RCall_orig/test/basic.jl RCall/test/basic.jl
```

Comment:
please add short description + author info?
(and don't forget to update the checksum :)